CHANGELOG.md

```diff
@@ -1,4 +1,436 @@
-# <v2.6.0> 2022-01-31
+# PySAL 23.01.0rc1 2023-01-23
+
+Overall, there were 479 commits that closed 205 issues, together with 19 pull requests since our last release on 2022-07-31.
+
+## Highlights
+
+- Version numbering has switched to YR.MO for the pysal meta-package release.
+- The `spopt` package added a LSCP-B formulation and the P-Dispersion model through the excellent GSoC project by [Erin Olson](https://erinrolson.github.io/2022/09/08/GSOC-fifth-blog.html)
+- `momepy` added enhanced approaches to network simplification through the GSoC project by [Greg Maya](https://github.com/gregmaya/gsoc2022_network_simpl) 
```

Comment:
```suggestion
```
we won't be able to cut 0.6, the documentation and some follow-up are not finished and I don't want to split it.